cmd/go-clang-gen/main.go

```diff
@@ -203,6 +203,32 @@ func prepareStructMembers(s *gen.Struct) {
 			}
 		}
 	}
+
+	// If the struct has two member variables, one is an array and the other a plain int/uint with size/length/count/len as its name we connect them two handle a slice.
+	if len(s.Members) == 2 {
+		if ((s.Members[0].Type.PointerLevel == 1 && s.Members[1].Type.PointerLevel == 0) || (s.Members[0].Type.PointerLevel == 0 && s.Members[1].Type.PointerLevel == 1)) &&
```

Comment:
The "isarrayLenKombo" makes sense but I would leave out the "isNeitherSliceNorArray" since the members already describe the meaning good enough